ThingIFSDK/ThingIFSDKTests/PostNewCommandWithCommandFormTests.swift

```diff
@@ -0,0 +1,364 @@
+    {
+        if actual.count != expected.count {
+            return false
+        }
+
+        for (key, value) in actual {
+            if (!value.isEqual(expected[key])) {
+                return false
+            }
+        }
+        return true;
+    }
+
+    struct TestCase {
+        let target: Target
+        let schemaName: String
+        let schemaVersion: Int
+        let actions: [Dictionary<String, AnyObject>]
+        let title: String?
+        let commandDescription: String?
+        let metadata: Dictionary<String, AnyObject>?
+        let issuerID: TypedID
+    }
+
+    func testPostCommandWithCommandFormSuccess() {
+        let setting = TestSetting()
+        let api = setting.api
+        let target = setting.target
+        let schema = setting.schema
+        let schemaVersion = setting.schemaVersion
+        let owner = setting.owner
+
+        // perform onboarding
+        api._target = target
+
+        let testCases = [
+            TestCase(target: target,
+                     schemaName: schema,
+                     schemaVersion: schemaVersion,
+                     actions: [["turnPower":["power": true]]],
+                     title: nil,
+                     commandDescription: nil,
+                     metadata: nil,
+                     issuerID: owner.typedID),
+            TestCase(target: target,
+                     schemaName: schema,
+                     schemaVersion: schemaVersion,
+                     actions: [["setBrightness":[
+                                 "brightness": 100]]],
+                     title: nil,
+                     commandDescription: nil,
+                     metadata: nil,
+                     issuerID: owner.typedID),
+            TestCase(target: target,
+                     schemaName: schema,
+                     schemaVersion: schemaVersion,
+                     actions: [["turnPower": ["power": true]],
+                         ["setBrightness": ["brightness": 100]]],
+                     title: nil,
+                     commandDescription: nil,
+                     metadata: nil,
+                     issuerID: owner.typedID),
+            TestCase(target: target,
+                     schemaName: schema,
+                     schemaVersion: schemaVersion,
+                     actions: [["turnPower":["power": true]]],
+                     title: "title",
+                     commandDescription: nil,
+                     metadata: nil,
+                     issuerID: owner.typedID),
+            TestCase(target: target,
+                     schemaName: schema,
+                     schemaVersion: schemaVersion,
+                     actions: [["turnPower":["power": true]]],
+                     title: nil,
+                     commandDescription: "command description",
+                     metadata: nil,
+                     issuerID: owner.typedID),
+            TestCase(target: target,
+                     schemaName: schema,
```

Comment:
I fixed according to this comment but each TestCases are very similar. So this comment is not removed.